tests/Unit/UserTest.php

```diff
@@ -87,7 +87,7 @@ public function testLastLogin()
 
         $last = $this->user->lastLogin();
 
-        $this->assertInstanceOf(Login::class, $last); // @phpstan-ignore-line
```

Comment:
Actions are failing since you removed this. Perhaps your local version of PHPStan needs to be updated, if it is reporting "no error to ignore"?

Reply:
You're right, I didn't update after the last devkit update.